lib/modules/dosomething/dosomething_signup/includes/Signup.php

```diff
@@ -110,7 +110,7 @@ private function build($data) {
       'id' => isset($northstar_user) ? $northstar_user->id : $data->field_northstar_id_value,
       'first_name' => dosomething_helpers_extract_field_data($drupal_user->field_first_name),
       'last_initial' => $last_initial,
-      'photo' => dosomething_helpers_extract_field_data($drupal_user->photo),
+      'photo' => isset($drupal_user->picture) ? $drupal_user->picture : NULL,
```

Comment:
@chloealee Yeah I think that's correct! I wonder if we could "merge" those two properties and just reformat the MCC campaign badge to use a rounded avatar instead...? It is pretty confusing to have both. 😕 

If we add the user property in the Northstar proxy, we can just remove the `$this->user` business in the `build` function and then "append" it to each signup in the result in the Northstar proxy method (since then it could happen as a single database query, rather than multiple HTTP requests).